content/Constants/Constants.q.md

```diff
@@ -0,0 +1,1453 @@
+# Constants
+
++++
+
+### Part 1: Sample Code Analysis
+
+:::
+
+/// type=REPL, readonly=true
+
+```php
+<?php
+class Circle
+{
+    const PI = 3.1416;
+    
+    public static function area($radius)
+    {
+        return self::PI * $radius * $radius;
+    }
+}
+
+echo "The PI value is: " . Circle::PI;
+?>
+```
+/// type=SS, answer=[5]
+
+Execute the program. What is its output?
+
+ - It prints `PI`.
+
+ - It prints `3.1416`.
+
+ - It prints `area()`.
+
+ - It prints `$radius`.
+
+ - It prints `The PI value is: 3.1416`.
+
+
+/// type=SS, answer=[2]
+
+Which of the following is a class?
+
+ - `PI`
+
+ - `Circle`
+
+ - `self::PI`
+
+ - `Circle::PI`
+
+ - `const PI = 3.1416;`
+
+
+/// type=SS, answer=[2]
+
+Which of the following is a static method?
+
+ - `PI`
+
+ - `area()`
+
+ - `Circle`
+
+ - `$radius`
+
+ - `Circle::PI`
+
+
+/// type=SS, answer=[3]
+
+In the statement `const PI = 3.1416;` on line 4, what is `const`?
+
+ - It is a class.
+
+ - It is a method.
+
+ - It is a keyword.
+
+ - It is a constant.
+
+ - It is a property.
+
+
+/// type=SS, answer=[5]
+
+In the statement `const PI = 3.1416;` on line 4, what is `PI`?
+
+ - It is a class.
+
+ - It is a method.
+
+ - It is a keyword.
+
+ - It is a property.
+
+ - It is a constant name.
+
+
+/// type=SS, answer=[2]
+
+In the statement `const PI = 3.1416;` on line 4, what does the `const` keyword do?
+
+ - It sets the `PI` class constant.
+
+ - It defines the `PI` class constant.
+
+ - It removes the `PI` class constant.
+
+ - It displays the `PI` class constant.
+
+ - It accesses the `PI` class constant.
+
+
+/// type=SS, answer=[4]
+
+Which statement best describes the code on line 4?
+
+ - It sets the value of the `PI` property to `3.1416`.
+
+ - It accesses the value of the `PI` constant in the `Circle` class.
+
+ - It assigns the value `3.1416` to the `PI` property of the `Circle` class.
+
+ - It defines the `PI` constant of the `Circle` class with the fixed value `3.1416`.
+
+ - It defines the `PI` property of the `Circle` class with the default value `3.1416`.
+
+
+/// type=SS, answer=[5]
+
+On line 8, what does `self::PI` do?
```

Comment:
Thank you for the suggestion. I will not do any changes to this one since the difference is not that visible and it might differ when rendered in the reader. Please check this. Thanks.
![image](https://user-images.githubusercontent.com/25945790/68755214-1863f180-0643-11ea-90c3-62fe4b6fc84d.png)